pycyphal/transport/udp/_udp.py

```diff
@@ -258,7 +242,8 @@ def output_sessions(self) -> typing.Sequence[UDPOutputSession]:
         return list(self._output_registry.values())
 
     @property
-    def local_ip_address(self) -> typing.Union[ipaddress.IPv4Address, ipaddress.IPv6Address]:
+    def local_ip_addr(self) -> typing.Union[ipaddress.IPv4Address, ipaddress.IPv6Address]:
+        assert isinstance(self._sock_factory, IPv4SocketFactory)
```

Comment:
polymorphism has left the chat

![image](https://user-images.githubusercontent.com/3298404/223409328-ff220dcf-679c-4742-918e-99caac742cb9.png)

Reply:
Also: `local_ip_addr` -- I sense a shortage of letters